.kodiak.toml

```diff
@@ -1,5 +1,12 @@
 version = 1
 
+[approve]
+auto_approve_usernames = ["dependabot"]
+
 [merge]
 method = "squash"
 automerge_label = "automerge 🚀"
+
+[merge.automerge_dependencies]
+versions = ["major", "minor", "patch"]
```

Comment:
As discussed, `major` is probably not a good idea for this repo

Reply:
That's fair; I've changed it. I'm not sure what extra manual validation reviewers are meant to be doing of major updates though.